### ISSUE TITLE
Doesn't Load in Nuke 12.1v2

### ISSUE THREAD

**Issue by jedypod** [adrianpueyo/KnobScripter#6]
It looks like there's an issue loading knob scripter in Nuke12.1v2. Traceback below:
```
Traceback (most recent call last):
  File "~/.nuke/Knob_Scripter/knob_scripter.py", line 3920, in showKnobScripter
    pan = KnobScripter(selection[0], knob)
  File "~/.nuke/Knob_Scripter/knob_scripter.py", line 99, in __init__
    self.nukeSEOutput = self.findSEOutput(self.nukeSE)
  File "~/.nuke/Knob_Scripter/knob_scripter.py", line 1644, in findSEOutput
    return se.children()[-1].children()[1]
IndexError: list index out of range
```
The behavior seems to be related to if there is a `ScriptEditorHistory.xml` file in the user's home directory.

Some cursory testing indicates that the issues doesn't recur if you disable `save and restore script editor history` in the Nuke preferences, and delete the `ScriptEditorHistory.xml` file.

I have no idea why this would affect KnobScripter's ability to find the script editor window among the qgwidgets... maybe you have an idea.

Hope this helps in tracking down the issue!

**Comment by jedypod** [comment]
After some investigation, it turns out this issue is caused by the code that attempts to locate the nuke script editor window. There were some changes made with the Qt port in nuke 12 that broke the way the Qt objects were being located.

I've created (a pull request)[https://github.com/adrianpueyo/KnobScripter/pull/7] which addresses this issue.

**Comment by Ripax** [comment]
yes iam also having same issue

**Comment by adrianpueyo** [comment]
Thanks! Addressed on KnobScripter 2.4 (currently testing on Develop).
